templates/state_machine.yaml

```diff
@@ -43,6 +44,10 @@ Parameters:
   JobTableName:
     Description: Table name for Jobs Table
     Type: String
+  LambdaJobsMemorySize:
+    Type: Number
+    MinValue: 128
+    MaxValue: 10240
```

Comment:
Nitpick: I don't think we need Min/Max validation here since it's done in the parent stack.

Reply:
making cfn-lint happy